ui/app/components/feedback/HumanFeedbackButton.tsx

```diff
@@ -1,14 +1,15 @@
-import { Button } from "~/components/ui/button";
+import { Button, type ButtonProps, ButtonIcon } from "~/components/ui/button";
 import { Feedback } from "../icons/Icons";
 
-export interface HumanFeedbackButtonProps {
-  onClick: () => void;
-}
+export type HumanFeedbackButtonProps = Omit<
```

Comment:
Expanding props here will be useful in a future PR where I'll likely change the composition for the modal.